api/mfx_dispatch/windows/include/mfx_dispatcher_defs.h

```diff
@@ -40,3 +40,8 @@
 typedef void * mfxModuleHandle;
 
 typedef void (MFX_CDECL * mfxFunctionPointer)(void);
+
+// Tracer uses lib loading from Program Files logic (via Dispatch reg key) to make dispatcher load tracer dll.
+// With DriverStore loading put at 1st place, dispatcher loads real lib before it finds tracer dll.
+// This workaround explicitly checks tracer presence in Dispatch reg key and loads tracer dll before the search for lib in all other places.
+#define MFX_TRACER_WA_FOR_DS 1
```

Comment:
I don't quite get the reason to have ever defined MFX_TRACER_WA_FOR_DS. If there is no way to redefine that, then what's the purpose?